code/datums/components/crafting/recipes.dm

```diff
@@ -475,15 +475,28 @@
 	category = CAT_WEAPONRY
 	subcategory = CAT_AMMO
 
-/datum/crafting_recipe/ishotgun
-	name = "Improvised Shotgun"
-	result = /obj/item/gun/ballistic/shotgun/doublebarrel/improvised
+/datum/crafting_recipe/musket
+	name = "Maintenance musket"
+	result = /obj/item/gun/ballistic/rifle/boltaction/musket
 	reqs = list(/obj/item/weaponcrafting/receiver = 1,
 				/obj/item/pipe = 1,
 				/obj/item/weaponcrafting/stock = 1,
 				/obj/item/stack/package_wrap = 5)
```

Comment:
I remember discussing some alternate recipes for improvised shotguns on the discord server a few weeks ago- being able to use a plumbing duct instead of an atmos pipe, being able to use tape or bedsheets instead of wrapping paper, etc.

Do those have a chance of making it in in this PR?

Reply:
Eh not really but this is a fantastic opportunity to replace paper wrapping with duct tape thanks for reminding me.